addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -1,20 +1,22 @@
-odoo.define('website_slides.quiz', function (require) {
-    'use strict';
+/** @odoo-module **/
 
-    var publicWidget = require('web.public.widget');
-    var Dialog = require('web.Dialog');
-    var core = require('web.core');
-    var session = require('web.session');
+    import publicWidget from 'web.public.widget';
+    import Dialog from 'web.Dialog';
+    import core from 'web.core';
+    import session from 'web.session';
 
-    var CourseJoinWidget = require('website_slides.course.join.widget').courseJoinWidget;
-    var QuestionFormWidget = require('website_slides.quiz.question.form');
-    var SlideQuizFinishModal = require('website_slides.quiz.finish');
+    import courseJoinWidget from '@website_slides/js/slides_course_join';
+    import QuestionFormWidget from '@website_slides/js/slides_course_quiz_question_form';
+    import SlideQuizFinishModal from '@website_slides/js/slides_course_quiz_finish';
 
-    var SlideEnrollDialog = require('website_slides.course.enroll').slideEnrollDialog;
+    import slideEnrollDialog from '@website_slides/js/slides_course_enroll_email';
 
     var QWeb = core.qweb;
     var _t = core._t;
 
+    const CourseJoinWidget = courseJoinWidget.CourseJoinWidget;
+    const SlideEnrollDialog = slideEnrollDialog.SlideEnrollDialog;
```

Comment:
```suggestion
    const { CourseJoinWidget } = courseJoinWidget;
    const { SlideEnrollDialog } = slideEnrollDialog;
```
(the best way would be to export those as named exports from the original module, this way you could directly import them from the module instead, but that more changes so feel free to not do that)